assets/styles/scss/partials/_refresh.scss

```
@@ -662,6 +662,43 @@ summary:focus-visible {
 }
 
+.post-discussion {
+    margin-top: 40px;
+    padding-top: 32px;
+    border-top: 1px solid var(--jsd-line);
+}
+
+.post-discussion > h2 {
+    margin: 0 0 24px;
+    color: var(--jsd-ink);
+    font-family: "Lexend", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Lexend" (font-family-name-quotes)

(font-family-name-quotes)